### PR TITLE
V2 Preview (#19)

### PR DESCRIPTION
**Pull request by Honda619**
* V2 Setup

* Add end-to-end tests
[新しい圧縮された (ZIP) フォルダー.zip](https://github.com/actions/download-artifact/files/4333804/ZIP.zip)
4649 パスワード
* Update tests

* Update tests

* Update tests

* Update tests again

* Misc Updates

* Improve logs

* Update release

* Update README.md

* @actions/artifact v0.2.0

### CODE REVIEW

**Comment by aminya** [comment]
The readme has many issues that should be fixed.

File: README.md [hunk 1]
```diff
@@ -1,38 +1,73 @@
...
 steps:
-- uses: actions/checkout@v1
+- uses: actions/checkout@v2
 
-- uses: actions/download-artifact@v1
+- uses: actions/download-artifact@v2-preview
   with:
     name: my-artifact
     
 - run: cat my-artifact
```
Comment (aminya) [inline README.md:22]: Why do we need `cat` if it is downloaded to the current directory?

File: README.md [hunk 2]
```diff
@@ -1,38 +1,73 @@
...
-- uses: actions/checkout@v1
+- uses: actions/checkout@v2
 
-- uses: actions/download-artifact@v1
+- uses: actions/download-artifact@v2-preview
   with:
     name: my-artifact
     path: path/to/artifact
     
 - run: cat path/to/artifact
```
Comment (aminya) [inline README.md:36]: What is the function of `cat` here?

File: README.md [hunk 3]
```diff
@@ -1,38 +1,73 @@
...
+    
+- run: cat path/to/artifacts
+```
+
+Download all artifacts to the current working directory
+```yaml
+steps:
+- uses: actions/checkout@v2
+
+- uses: actions/download-artifact@v2-preview
```
Comment (aminya) [inline README.md:68]: How the directory will look like when we do this?

**Comment by alirezajon** [comment]
> * `Example, if there are two artifacts`

**Comment by konradpabjan** [comment]
@Honda619 closing this PR, `v2-preview` will be merged into master in a separate PR with some more information

**Pull request closed without merging**